### PR TITLE
Set the timeout for ES request handling to 10 mins to avoid "Failed t…

### PR DESCRIPTION
**Pull request by PremyslCerny**
…o get data from Elasticsearch" message in UI

### CODE REVIEW

**Comment by eliska-n** [comment]
oh, I'm sorry, I completely forgot to follow MRs here. 

I'm fine with that. But... who is able to wait 10 mins for the screen to load? :D such request would be probably interrupted by the user.

File: Site/ASAB Maestro/Descriptors/elasticsearch.yaml [hunk 1]
```diff
@@ -50,6 +50,12 @@ nginx:
       - auth_request_set $batman $upstream_http_authorization
       - proxy_set_header Authorization $batman
 
+      # The timeout for ES request handling is set to 10 mins to avoid "Failed to get data from Elasticsearch" message in UI
+      - proxy_connect_timeout 600;
```
Comment (eliska-n) [inline Site/ASAB Maestro/Descriptors/elasticsearch.yaml:54]: the syntax is wrong. There should not be the `;` in the end

**Review by eliska-n — CHANGES_REQUESTED** [review]

**Comment by eliska-n** [comment]
Maestro is on app.logman right now. I can't approve MRs to be tested from master anymore. It could break app.logman. It has to be tested before merging.

**Comment by ateska** [comment]
@PremyslCerny ?

**Comment by PremyslCerny** [comment]
It is deployed in nginx in Prima and Composite. If you want me to test it with Maestro, you need to show me how to do it properly, maybe on some WS.

**Review by ateska — APPROVED** [review]

**Pull request merged**